<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>조건 등록</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>701</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>단일종목</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>461</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>종목</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>111</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>종목 이름</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="tv_codeName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>종목 코드</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="et_code">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_searchCode">
      <property name="geometry">
       <rect>
        <x>391</x>
        <y>20</y>
        <width>61</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>검색</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>461</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>매수 조건</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>77</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>총금액</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>77</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>매수가</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="et_buyPrice">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>24</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="et_totalBuyAmout">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>54</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>77</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>시간</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="et_buyStartTime">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>85</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="date">
       <date>
        <year>1999</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="time">
       <time>
        <hour>9</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="displayFormat">
       <string>AP h:mm</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="et_buyEndTime">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>84</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="date">
       <date>
        <year>1999</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="time">
       <time>
        <hour>15</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>1999</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>AP h:mm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>287</x>
        <y>88</y>
        <width>21</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>~</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>550</y>
       <width>291</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>*최대 손/익절율 도달 시 전량 매도</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>461</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>손절 조건</string>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>부분 손절수량</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>부분 손절율</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="et_lossRate">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="et_lossRateVolume">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>최대 손절율</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="et_maxLossRate">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>90</y>
        <width>41</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>(%)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>41</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>(%)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>41</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>(%)</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>461</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>익절 조건</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>부분 익절수량</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>부분 익절율</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="et_profitRate">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="et_profitRateVolume">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>최대 익절율</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="et_maxProfitRate">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>90</y>
        <width>41</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>(%)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>41</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>(%)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>41</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>(%)</string>
      </property>
     </widget>
    </widget>
    <widget class="QDialogButtonBox" name="bts_oneStock">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>630</y>
       <width>341</width>
       <height>32</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>570</y>
       <width>441</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>*모든 항목은 필수 입력 사항</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>키움조건</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bts_oneStock</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bts_oneStock</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
